addon/components/file-card.hbs

```diff
@@ -0,0 +1,38 @@
+<div
```

Comment:
This should ideally be moved to appuniversum. This isn't strictly related to the form fields, so it feels out of place in this addon.

Reply:
https://github.com/appuniversum/ember-appuniversum/issues/213